README.md

```diff
@@ -90,7 +90,7 @@ you prefer to only receive updates for tagged releases, use
 
 ## Contributing
 
-See the [CONTRIBUTING.md][] document.
```

Comment:
`[CONTRIBUTING.md][]` is invalid Markdown. It renders as:

![image](https://cloud.githubusercontent.com/assets/18138/13933234/61e6dacc-efac-11e5-88dc-b72f3ce27ac3.png)

So if you want to keep it, then it should be `[CONTRIBUTING.md](CONTRIBUTING.md)`